docs/README.md

```diff
@@ -46,6 +46,67 @@ An app that makes it possible to render external iframes on a store
 | `width` | Number | Width attribute of the iframe | `null`
 | `height` | Number | Height attribute of the iframe | `null`
 
+## Configuration - dynamic Iframe
+
+1. Add the `vtex.iframe` to the theme's dependencies on the `manifest.json`
+```
+...
+"dependencies": {
+ ...
+ "vtex.iframe": "0.x"
+ ...
+}
+...
+```
+ 
+ 2. Add the interface `dynamicIframe` to any **custom page** (Dynamic Iframes are not allowed outside custom pages).
+ 
+ ```
+ ...
+"store.custom.locationPage":{
+        "allowed":[
+            "dynamicIframe"
```

Comment:
The interface could be an extension of the `iframe`, like: `iframe.dynamic-src`

With that it would be possible to use this block anywhere `iframe` is already allowed.

What do you think?

Reply:
great suggestion, do you have an example I could follow for making that change?

Reply:
I'll show you how to do this @mvandeusen